client/src/screens/availability/Edit.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+import { Button, ButtonRow } from '../../components/Button';
+import { DatePicker } from '../../components/Calendar';
+import { Container, Holder } from '../../components/Container';
+import { Separator } from '../../components/Separator';
+
+import { addAvailability } from '../../state/availability.actions';
+
+class Edit extends Component {
+  static navigationOptions = () => ({
+    title: 'Availability',
```

Comment:
`'Edit Availability'`?

Reply:
If you change this - it will change the title of the tabIcon in the tabBar - I'll look into a work around.